recipes-wpe/wpeframework/include/cobalt.inc

```diff
@@ -24,9 +30,16 @@ WPE_COBALT_EXTRAFLAGS = ' \
     -DPLUGIN_COBALT_GROUP=${WPE_COBALT_GROUP} \
     -DWPEFRAMEWORK_PLUGIN_COBALT_STARBOARD_CONFIGURATION_INCLUDE=${WPE_COBALT_STARBOARD_CONFIGURATION_INCLUDE} \
     -DPLUGIN_COBALT_RESOLUTION=${WPE_COBALT_RESOLUTION} \
+    -DPLUGIN_COBALT_CLIENTIDENTIFIER=${WPE_COBALT_CLIENTIDENTIFIER} \
+    -DPLUGIN_COBALT_MANUFACTURER_NAME=${WPE_COBALT_MANUFACTURER_NAME} \
+    -DPLUGIN_COBALT_CHIPSET_MODEL_NUMBER=${WPE_COBALT_CHIPSET_MODEL_NUMBER} \
     -DPLUGIN_COBALT_FIRMWARE_VERSION=${WPE_COBALT_FIRMWARE_VERSION} \
     -DPLUGIN_COBALT_MODEL_NAME=${WPE_COBALT_MODEL_NAME} \
     -DPLUGIN_COBALT_MODEL_YEAR=${WPE_COBALT_MODEL_YEAR} \
+    -DPLUGIN_COBALT_OPERATOR_NAME=${WPE_COBALT_OPERATOR_NAME} \
+    -DPLUGIN_COBALT_FRIENDLY_NAME=${WPE_COBALT_FRIENDLY_NAME} \
+    -DPLUGIN_COBALT_SCOPE=${WPE_COBALT_SCOPE} \
+    -DPLUGIN_COBALT_SECRET=${WPE_COBALT_SECRET} \
```

Comment:
Can you remove scope and secret?, because they will not come from configuration.

Reply:
Removed